### PR TITLE
Added how to guide girls in india

### PR DESCRIPTION
**Pull request by ankit071105**
# Added how to guide girls in india

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,30 @@
----
-layout: default
-description: "Learn how to make your first web app with Ruby on Rails! Sign up for a Rails Girls workshop in your area and follow these guides."
----
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (runlevel5) [inline index.html:5]: Why do you introduce these additional tags? It's also wrong.

Comment (ankit071105) [reply index.html:5]: okay sorry for that

File: index.html [hunk 2]
```diff
@@ -20,6 +43,7 @@ <h2>Guides in other languages</h2>
       <li><a href="https://railspremierspas.humancoders.com/">Guides in French</a></li>
       <li><a href="https://rgcn.github.io">Guides in Chinese</a></li>
       <li><a href="https://rgua.github.io">Guides in Russian</a></li>
+      <li><a href="india.html">Guides in Indian</a></li>
```
Comment (runlevel5) [inline index.html:46]: This should be a link to a fully translated version in another language. In your case I believe it should be Hindi or another language that you want to support.

I have a look into the `india.html` and all I see is an HTML with some objectives written in English and has no translated text of the guides itself.

**Comment by subhasish-roy** [comment]
hello runlevel5, i wish to add hindi language to the webiste but i am a beginner so can you please just guide me on what are the steps involved in it for doing it?

**Pull request closed without merging**